### DISCUSSION TITLE
Q: Will it be possible to prompt the user while running the ansible playbook?

### DISCUSSION THREAD

**Discussion by Tebra** [semaphoreui/semaphore#1770] [Q&A]
I really appreciate what you did with the semaphore ui. Love that you used vue for it! 

My question is if you would consider adding the possibility to prompt for confirmation or input of the user depending on the result of an ansible task. 

This would enable many of general, guided provisioning processes.

**Comment by aric89** [comment]
I think you are looking for this.

Survey Variables allow you to ask the task runner for mandatory or optional variables to pass. The name is the variable name that will be passed to Ansible. For example, if you set the name as survey_username, it can then be accessed in your script as {{survey_username}}. The title is what the task runner will see in the GUI.

**Comment by Creamers158** [comment]
The idea is for example using `--step` argument with `serial: 1`
Running a playbook on many nodes one by one and confirming before proceding the next run on the next host.

So interaction to confirm would be great
